guava/src/com/google/common/reflect/ClassPath.java

```diff
@@ -296,7 +296,7 @@ public String getSimpleName() {
         String innerClassName = className.substring(lastDollarSign + 1);
         // local and anonymous classes are prefixed with number (1,2,3...), anonymous classes are
         // entirely numeric whereas local classes have the user supplied name as a suffix
-        return CharMatcher.digit().trimLeadingFrom(innerClassName);
+        return CharMatcher.inRange('0', '9').trimLeadingFrom(innerClassName);
```

Comment:
Similar question.
As well can you add a UT for this?

Reply:
There is already UT for both of the classes. As I am not adding extra logic, I didn't add any and there were no failures during compilation.